### PR TITLE
Adding a Cypress test for the amp-img

### LINKED ISSUE #953: Integrate AmpImg component
**Is your feature request related to a problem? Please describe.**
Depends upon https://github.com/bbc/psammead/pull/241 (to do) and #927 (done).

Once we have our Container and Presentational components, we can integrate into the renderer.

**Describe the solution you'd like**
Using the `PlatformContext` added in #982, decide which Image to render (either the canonical or AMP version - exposed by https://github.com/bbc/psammead/pull/241).

**Testing notes**
Acceptance criteria:

- We should be able to view an AMP version of an article and inspect any image in that article and find that it is using `<amp-img>` rather than ``.

**Additional context**
#442 is blocked until this is done.

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by jamesbrumpton**
Resolves #953 

_This adds a Cypress test to make sure that the image that is rendered on the page is an `amp-img`._

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by jamesbrumpton** [comment]
I've put the blocked label onto this until #1153 has been merged as this test will fail otherwise.

**Review by dr3 — DISMISSED** [review]

File: cypress/integration/ampSpec.js
Comment (ChrisBAshton) [inline cypress/integration/ampSpec.js]: Hmmm does this make an assertion? `getElement('amp-img')` could return null but as we're not checking its value, the tests would pass.

```suggestion
      getElement('amp-img').should('be.visible');
```

**Review by ChrisBAshton — COMMENTED** [review]
Just added a comment to clarify something

**Comment by bcmn** [comment]
No longer blocked on #953.

**Review by ChrisBAshton — APPROVED** [review]
Awesome, thanks 👍

**Review by dr3 — APPROVED** [review]

**Pull request merged**
